src/components/flat-table/flat-table-test.stories.mdx

```diff
@@ -15,7 +15,8 @@ import {
 } from ".";
 import guid from "../../__internal__/utils/helpers/guid";
 import { FLAT_TABLE_SIZES, FLAT_TABLE_THEMES } from "./flat-table.config";
-import { FlatTableSortableStory } from "./flat-table.stories.mdx";
+import { FlatTableSortableStory } from "./flat-table.stories.mdx"
```

Comment:
I think we have decided that having `test` stories in MDX is a bad idea and we're slowly rewriting them to be `.js` again, if it's not a problem could you please do it in this PR?